src/crimson/os/seastore/onode_manager/staged-fltree/fltree_onode_manager.h

```diff
@@ -78,27 +80,142 @@ struct FLTreeOnode final : Onode, Value {
     return *read_payload<onode_layout_t>();
   }
 
-  onode_layout_t &get_mutable_layout(Transaction &t) final {
+  template <typename layout_func_t, typename recorder_func_t>
+  void with_mutable_layout(
+    Transaction &t,
+    layout_func_t &&layout_func,
+    recorder_func_t &&recorder_func) {
```

Comment:
nit, looks `layout_func_t` and `recorder_func_t` can be merged into 1 function.

Reply:
Done.